generated/nidaqmx/_lib.py

```diff
@@ -146,29 +162,25 @@ def _import_lib(self):
         windll = None
         cdll = None
 
-        if sys.platform.startswith('win') or sys.platform.startswith('cli'):
-            if 'iron' in platform.python_implementation().lower():
-                windll = ctypes.windll.nicaiu
-                cdll = ctypes.cdll.nicaiu
-            else:
-                windll = ctypes.windll.LoadLibrary('nicaiu')
-                cdll = ctypes.cdll.LoadLibrary('nicaiu')
-
+        if sys.platform.startswith('win'):
```

Comment:
FYI: the "cli" check was for IronPython. 
https://www.github.com/IronLanguages/ironpython3/issues/395

Reply:
Ah, good to know, thanks!